recipes/recipes_emscripten/lfortran/build.sh

```diff
@@ -0,0 +1,21 @@
+set -e
+set -x
+
+mkdir -p src/bin/asset_dir
```

Comment:
For building LFortran to wasm, we build it twice. First is a regular build which is used to compile the pure runtime library (consisting of fortran files) to `mod` files. We put these `mod` files into the `src/bin/asset_dir`. This way they can now be packed along with LFortran while building it to `wasm` (Second build).